terraform/deployments/govuk-publishing-platform/backends_origin.tf

```diff
@@ -130,6 +131,13 @@ resource "aws_lb_target_group" "publisher" {
   health_check {
     path = "/healthcheck"
   }
+
+  tags = merge(
+    local.additional_tags,
+    {
+      name = "publisher-tg"
```

Comment:
Is it intentional that some of these are lowercase `name` as opposed to `Name`? (the latter being the magic tag that the AWS console looks for as a display name for the resource, in some cases - if I understand correctly)

Reply:
Yes it was intentional - as it was a topic I wanted to discuss but slipped my mind, having "Name" in fact does get used to populate the name field  - an example of that would be Security groups and ACM, for ACM I actually did use capital "Name" for that reason.
I am happy to update to name to fill the name fields where they are missing.

Reply:
No probs. Are there any situations where we would want to use `name` then? Should we standardise on `Name` everywhere?

Reply:
There can be as Name and name will be seperate Tags, for example, name could be used for cost allocation reason whereas Name would not, with both also containing different Value.
In our case it should be no need - so changing to Name will be best practice.